teamcity.sh

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 set -e
 cp teamcity-settings.json src/montagu-deploy.json
-cd src
-pip3 install -r requirements.txt
-./deploy.py
-./test.py $@
\ No newline at end of file
+pip3 install --quiet -r src/requirements.txt
+./src/deploy.py
+./src/test.py $@
```

Comment:
Need to run `deploy.py` as `./src/deploy.py` or it can't find the submodules. Also, this is how we always run it for deployments, so TeamCity should be doing the same thing as us!